src/components/DashboardResultIndicatorStatistics.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div class="progressStatistics">
+    <div>
+      <span class="progressStatistics__title">
+        Siste verdi
+        <template v-if="latestProgressRecord">
+          per {{ formatDate(latestProgressRecord.timestamp) }}
+        </template>
+      </span>
+      <div>
+        <span class="progressStatistics__value">
+          <template v-if="resultIndicator && latestProgressRecord">
+            {{ formatKPIValue(resultIndicator, latestProgressRecord.value) }}
+            <span v-if="periodTrend" :class="bgColor" class="progressStatistics__trend">
+              {{ periodTrendFormatted }}
+            </span>
+          </template>
+          <template v-else>–––</template>
+        </span>
+      </div>
+    </div>
+    <div>
+      <span class="progressStatistics__title">
+        {{ $t('kpi.goals.goal') }}
+        <template v-if="resultIndicator && goal">
+          for {{ formatDateRange(goal.fromDate, goal.toDate) }}
+        </template>
+      </span>
+      <div>
+        <span class="progressStatistics__value">
+          <template v-if="resultIndicator && goal">
+            {{ formatKPIValue(resultIndicator, goal.value) }}
+            <icon-alert-success v-if="isGoalReached" :width="20" :height="20" />
+          </template>
+          <template v-else>–––</template>
+        </span>
+      </div>
+    </div>
```

Comment:
Kanskje bare droppe hele denne hvis ikke det finnes noe mål?

Reply:
👍  Dropper begge statsene når det ikke er noe å vise (men beholder høyden på raden for å unngå mye hopping) ved skifte av RI.